app/assets/javascripts/backbone/views/timeline_select_steps_view.coffee

```diff
@@ -1,6 +1,6 @@
 class MS.Views.TimelineSelectStepsView extends Backbone.View
 
-  el: 'body'
+  el: '#timeline-header'
```

Comment:
`#timeline-header` includes not just the header, but the whole timeline-list. Similar for `#timeline-table` on the other page.